.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 python:
-  - "2.7"
-  - "pypy"
+  - "3.7"
```

Comment:
Are we actually _deprecating_ 2.x support?

Reply:
I'll update the setup.py. I don't want to support 2.x moving forward.